data/schema/v2/Decision_Table-2-0-0.schema.json

```diff
@@ -0,0 +1,107 @@
+{
+  "title": "SsvcDecisionPoint",
```

Comment:
This seems to the wrong file? It is not the Decision Table Schema -

Reply:
It will be good to fix this - although the automation might generate it too.

Reply:
Yeah. It works better when we use the right object name 😁